covid19-dashboard/src/Types.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+
+// These are commonly-used types.
+export type ValueType = {[key: string]: number | string |  boolean | ValueType | any};
```

Comment:
value type is recursive? also, try to avoid using any here, which essentially removes all the benefits of typing...

Reply:
fixed.